src/content_delegate.js

```diff
@@ -414,10 +414,20 @@ ContentDelegate.prototype.showPdfPage = function(
     };
     history.replaceState({content: previousPageHtml}, '');
 
+    // store the result in chrome local storage
+
+    const nonce = "blob_upload_storage"
+    const data = { [nonce]: ab }
```

Comment:
I'm a little perplexed at two things:

1. Maybe this is a JS thing, but why do you pass `[nonce]` as the key instead of just using `nonce`? 

2. `nonce` feels like the wrong variable name. Maybe `file_storage_key` or something like that?

Reply:
I'm agnostic w/r/t name - I started with a nonce when I thought about each blob being stored with a unique key, but then moved to the fixed string key. 

In JS you normally couldn't set a key dynamically simply by passing in the variable name (I believe it's because if you use `{nonce: data }` the key will be `nonce` and not whatever the variable is). Now with ES6, you can use computed property names by enclosing the variable key in brackets. See "new notations" section of the [Object Initializer Docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer)

Reply:
Another comment on 1 - you could remove the variable all-together since it's a fixed string, but the only downside to that is that we currently pass the nonce (or whatever its called) to the uploadDocument function. So all the tests/related functions would need to be altered ... figured we could keep the variable, pass it, and then have the infrastructure to use actual dynamic keys later.

Reply:
Got it. Merging, but with the understanding that we may have more testing to do.